stable/nats/values-production.yaml

```diff
@@ -0,0 +1,241 @@
+## Bitnami NATS image version
+## ref: https://hub.docker.com/r/bitnami/nats/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/nats
+  tag: 1.1.0
+  pullPolicy: IfNotPresent
```

Comment:
Since a rolling tag is used, the image.pullPolicy should be Always.

Reply:
I'm okey with using `Always` on the `values.yaml`, but.. this `values-production.yaml` is supposed to be for production environments where I'm not sure if an update of the image should be done without the admin consent.

Reply:
the application version is always the same, the only change in the image is that we have updated the system packages to fix possible security issue so I don;t think it is an issue using also `Always` in the values-production.yaml.

Reply:
🤔 ...since this Docker images just packages a Golang binary and, as @tompizmor mentioned, the system packages shouldn't break anything, I'm okey with using `Always`